client/src/components/reviews/reviewListComponents/ReviewTiles.jsx

```diff
@@ -1,86 +1,9 @@
 import React from 'react';
-import {ReviewTileContainer, RecommendationContainer, SellersResponse} from '../sharedStyles/sharedStyledComponents';
-const {Checkmark} = require('react-checkmark');
+import {IndividualTile} from './IndividualTile.jsx';
```

Comment:
this was from nick adams' modal right?

Reply:
Actually this was me moving my files from ReviewTiles into IndividualTile. I was mapping twice in the one file and it was causing issues, so I had to move them so there was only one map function per file.